packages/flutter_test/test/widget_tester_leaks_test.dart

```diff
@@ -0,0 +1,203 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
+
+import 'utils/leaking_classes.dart';
+
+late final String _test1TrackingOnNoLeaks;
+late final String _test2TrackingOffLeaks;
+late final String _test3TrackingOnLeaks;
+late final String _test4TrackingOnWithCreationStackTrace;
+late final String _test5TrackingOnWithDisposalStackTrace;
+late final String _test6TrackingOnNoLeaks;
+late final String _test7TrackingOnNoLeaks;
+late final String _test8TrackingOnNotDisposed;
+
+void main() {
+  collectedLeaksReporter = (Leaks leaks) => verifyLeaks(leaks);
+  LeakTesting.settings = LeakTesting.settings.copyWith(ignoredLeaks: const IgnoredLeaks(), ignore: false);
```

Comment:
This line with the three "ignores" in it (`ignoredLeaks`, `IgnoredLeaks`, `ignore`) reads rather cryptic. I tried to figure out from the docs what this is all supposed to mean and I am left to wonder:

- why do we need to pass in empty IgnoredLeaks sets? If somebody has globally configured to ignore a certain leak (e.g. because something deep inside the framework is known to leak and we haven't cleaned that up yet), why does this test want to re-enable those leaks? Ideally, it would only track the leaks it actively creates itself, not any other leaks that on a global level were turned off? Or am I misunderstanding this?

- Should the "ignore: false" parameter be renamed to "enabled" to reflect (at least according to the docs) that it controls whether leak tracking is enabled or not? This also helps with not overusing the term "ignore" even more.

Reply:
In this exact test we want to test leak tracking works, not to verify leaks. So, we wipe off all settings.
But, you are right: some classes may be excluded when leaking and we do not want this test to fail.

We discussed naming a lot with Kenzie and agreed to use the same word everywhere for consistency.